site/guide/9-configuration.md

```diff
@@ -358,6 +363,25 @@ mutual TLS.
 
 [`mtls::Certificate`]: @api/rocket/mtls/struct.Certificate.html
 
+### Proxied TLS
+
+If Rocket is running behind a reverse proxy that terminates TLS, it is useful to
+know whether the original connection was made securely. Therefore, Rocket offers
+the option to configure a `proxy_proto_header` that is used to determine if the
+request is handled in a secure context. The outcome is available via
+[`Request::context_is_likely_secure()`] and used to set cookies' secure flag by
+default. To enable this behaviour, configure the header as set by your reverse
+proxy. For example:
```

Comment:
Again, we should be more specific: why is it useful? I'd suggest inverting this paragraph to something like:

```suggestion
The `proxy_proto_header` configuration parameter allows Rocket applications to
determine when and if a client's initial connection was likely made in a
secure context by examining the header with the configured name. The
header's value is parsed into a [`ProxyProto`] value and retrievable via
[`Request::proxy_proto()`].

That value is in-turn inspected to determine if the initial protocol was secure
(i.e, over TLS) and the result made available via
[`Request::context_is_likely_secure()`]. The value returned by this method
influences cookie defaults. In particular, if the method returns `true` (the
request context is likely secure) secure, the cookie `Secure` cookie flag is set
by default.
```